jcl/src/java.base/share/classes/java/lang/J9VMInternals.java

```diff
@@ -338,7 +338,11 @@ private static Throwable copyThrowable(Throwable throwable, HashMap hashMapThrow
 	}
 
 	/**
-	 * Private method to be called by the VM after a Threads dies and throws ThreadDeath
+/*[IF JAVA_SPEC_VERSION < 20]
+	 * Private method to be called by the VM after a Threads dies and throws ThreadDeath.
+/*[ELSE]
+	 * Private method to be called by the VM after a Threads dies.
+/*[ENDIF]
```

Comment:
Normally, I would ask that you repeat the `IF` condition on `ELSE` and `ENDIF`, but it may be more awkward than it's worth in javadoc comments; please fix lines 650 and 661 though.

Reply:
It seems more noise than useful when there is only a single line (or a few lines) inside the statements, but I've updated those lines.